hoomd/md/ForceCompositeGPU.h

```diff
@@ -68,11 +68,6 @@ class PYBIND11_EXPORT ForceCompositeGPU : public ForceComposite
 
     GPUArray<uint2> m_flag; //!< Flag to read out error condition
 
-    /// Number of rigid bodies on the local rank.
```

Comment:
`checkParticlesSorted` should also be removed above, as it has now gone to the CPU class.